src/puppetlabs/jdbc_util/pool.clj

```diff
@@ -18,6 +23,7 @@
     :driver-class-name (.setDriverClassName config value)
     :auto-commit (.setAutoCommit config value)
     :connection-timeout (.setConnectionTimeout config value)
+    :connection-check-timeout (add-connectivity-check-timeout-ms config value)
```

Comment:
i always feel weird about renaming things like this, though i guess it does fit better with connection-timeout